ops/main.py

```diff
@@ -174,7 +174,10 @@ def main(charm_class):
         juju_event_name = '{}_action'.format(juju_event_name)
 
     model_backend = ops.model.ModelBackend()
-    setup_root_logging(model_backend)
+    debug_stream = None
+    if 'JUJU_DEBUG' in os.environ:
```

Comment:
Should we check its value (can it be both 1 or 0)?

Reply:
it is actually set to a path where the execution for the debug session exist ("/tmp/juju-debug-XXXX/"). So it isn't a simple boolean, it is just a variable I'm sure is only set during debug-hooks or debug-code.
I had been doing:
`if os.getenv('JUJU_DEBUG'):`

which would cater for JUJU_DEBUG='', though Juju itself would never set it to that.